packages/browser-wallet/src/popup/shared/TransactionReceipt/displayPayload/DisplayDeployModule.tsx

```diff
@@ -18,8 +19,11 @@ export default function DisplayDeployModule({ payload }: Props) {
     return (
         <>
             <OptionalField title={t('version')} value={payload.version} />
-            <h5>{t('sourceHash')}:</h5>
-            <div className="mono transaction-receipt__value text-center">{chunkString(hash, 32).join('\n')}</div>
+            <h5>{t('moduleReference')}:</h5>
+            <div className="flex align-center">
+                <p className="mono transaction-receipt__value text-center m-0">{chunkString(hash, 32).join('\n')}</p>
+                <CopyButton className="transaction-receipt__copy-button" value={hash} />
```

Comment:
The copy button is almost on top of the hash. Perhaps the button should be below, or at least have some margin to separate them.

Reply:
I've ensured there is a gap now.

Reply:
Also made the text more centered (instead of both it and the button is centered)